src/panels/config/users/ha-config-users.ts

```diff
@@ -43,36 +43,73 @@ export class HaConfigUsers extends LitElement {
           ),
           sortable: true,
           filterable: true,
+          width: "25%",
           direction: "asc",
           grows: true,
           template: (name) => html`
             ${name ||
             this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
           `,
         },
+        username: {
+          title: this.hass.localize(
+            "ui.panel.config.users.picker.headers.username"
+          ),
+          sortable: true,
+          filterable: true,
+          width: "20%",
+          direction: "asc",
+          grows: true,
+          template: (username) => html`
+            ${username ||
+            this.hass!.localize("ui.panel.config.users.editor.unnamed_user")}
+          `,
+        },
         group_ids: {
           title: this.hass.localize(
             "ui.panel.config.users.picker.headers.group"
           ),
           sortable: true,
           filterable: true,
-          width: "30%",
+          width: "20%",
           template: (groupIds) => html`
             ${this.hass.localize(`groups.${groupIds[0]}`)}
           `,
         },
+        is_active: {
+          title: this.hass.localize(
+            "ui.panel.config.users.picker.headers.is_active"
+          ),
+          type: "icon",
+          width: "10%",
+          sortable: true,
+          filterable: true,
+          template: (is_active) => html`
+            ${is_active ? html` <ha-icon icon="hass:check"> </ha-icon> ` : ""}
+          `,
+        },
+        is_owner: {
```

Comment:
Feels weird to have a whole column just for a single user. We should override the group value with "Owner", since the user has all access regardless of the group.

Reply:
Override purely in the frontend rendering or in the backend service?

Reply:
Frontend.

Reply:
Okay, added that.